src/components/common/Dialogs/EditDomainDialog/EditDomainDialog.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Id } from '@colony/colony-js';
+import { string, object, number, boolean, InferType } from 'yup';
+
+import Dialog, { DialogProps, ActionDialogProps } from '~shared/Dialog';
+import { ActionHookForm as Form } from '~shared/Fields';
+
+import { ActionTypes } from '~redux/index';
+import { WizardDialogType } from '~hooks'; // useEnabledExtensions
+import { pipe, withMeta, mapPayload } from '~utils/actions';
+import { graphQlDomainColorMap } from '~types';
+import { DomainColor } from '~gql';
+import { notNull } from '~utils/arrays';
+
+import EditDomainDialogForm from './EditDomainDialogForm';
+import { getEditDomainDialogPayload } from './helpers';
+
+interface CustomWizardDialogProps extends ActionDialogProps {
+  filteredDomainId?: number;
+}
+
+type Props = DialogProps &
+  Partial<WizardDialogType<object>> &
+  CustomWizardDialogProps;
+
+const displayName = 'common.EditDomainDialog';
```

Comment:
I can't remember from a previous review if these displayNames are supposed to include `common.Dialogs...`?

Reply:
I don't think so, that would mean that there's a `Dialogs` component when that's just a containing folder